standard/mock/mock_team_template.lua

```diff
@@ -0,0 +1,256 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Mock/TeamTemplate
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- Warning: This mock ignores the posibility of image/imagedark being unset/empty
+-- and never uses legacyimage/legacyimagedark for that reason
+
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local MockData = Lua.import('test_assets.team_template_data')
+
+local NOT_FOUND_ERROR = '<div class="error">No team template exists for name "${name}".</div>'
+local MISSING_INPUT_ERROR = 'bad argument #1 to "name" (string or number expected, got nil)'
+
+local mockTeamTemplate = {}
+
+-- temp copy the orig functions so we can retrieve them later again when tearing down the mock
+local _teamTemplate = Table.copy(mw.ext.TeamTemplate)
```

Comment:
isn't the sense of this that we cache the orig functions so we can retrieve them when tearing the mock down again?
if we just set it as = instead of copy it would just overwrite them when we call setUp, or am i missing something?